logs/app.log

```diff
@@ -0,0 +1,16 @@
+2021-05-10 19:27:48,419 INFO mate.academy.service.AuthenticationServiceImpl:13 [main] method login() was called. Params: Login = bob
```

Comment:
same here - there is no need to push logs to github
you can add this folder to .gitignore as well